src/EventHub/EventHub/EventHub.csproj

```diff
@@ -11,7 +11,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.EventHub" Version="2.4.0" />
+    <Compile Remove="Cmdlets\NetworkRuleSet\NewAzureEventHubVNetRule.cs" />
```

Comment:
@v-Ajnava any reason that we're removing these files in the `.csproj` file and not just removing them in general?

Reply:
This was the old implementation of cmdlet